packages/archived/error-handling-react/package.json

```diff
@@ -55,11 +55,11 @@
     "@babel/preset-env": "^7.9.6",
     "@babel/preset-react": "^7.9.4",
     "@svgr/cli": "^5.3.1",
-    "@testing-library/react": "^10.4.4",
+    "@testing-library/react": "^14.0.0",
```

Comment:
revert all changes to this pkg, we dont maintain it, and i dont want history to make it look like we do

Reply:
reverted